source/layouts/layout.erb

```diff
@@ -299,34 +433,102 @@ under the License.
           <div class="container -wide">
             <div class="siteFooter__row">
               <div class="siteFooter__secondary__col-wrapper">
-                <div class="siteFooter__secondary__col siteFooter__secondary__col--terms">
-                  <a href="https://asana.com/terms" title="Terms &amp; Privacy" class="-white">
-                  Terms &amp; Privacy
+                <div class="siteFooter__secondary__col">
+                  <span title="Copyright">
+                    &copy;
+                    <script>
+                      document.write(new Date().getFullYear());
```

Comment:
Pretty neat lol. For some reason I always thought we'd hard-code the year (though now I realize that wouldn't be ideal at all)